lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -241,6 +241,11 @@
             <h2 class="heading -emphasized"><span><?php print t('Step 3: Do It'); ?></span></h2>
           </div>
         </div>
+        <?php if (!empty($partner_embed_markup)): ?>
```

Comment:
Should this be `!isset()`? If the partner embed isn't enabled, then the variable is never set.